### DISCUSSION TITLE
Campaign monitor - sync date fields

### DISCUSSION THREAD

**Discussion by anchovy** [solspace/craft-freeform#353] [Feature Requests]
Currently Campaign Monitor only syncs the following fields [#](https://docs.solspace.com/craft/freeform/v3/integrations/email-marketing/campaign-monitor.html): 

Text
Number
Multiple Options (can only select one)
Multiple Options (can select many)

Can we please get it to sync date fields / it's a mayor question on our forms for a site in development and is essential for us.

PHP version	8.0.9
Database driver & version	MySQL 8.0.26
Craft edition & version	Craft Pro 3.7.42
Yii version	2.0.45

Freeform (pro)	3.13.9

**Comment by gluehq** [comment]
Yes please this would be amazing. Campaign Monitor is the best and it would be a shame to switch away.

**Comment by anchovy** [comment]
@kjmartens - any thoughts on this? The CM integration seems to have the least field support out of all the Email Marketing Integrations.

**Comment by kjmartens** [comment]
I'll have a developer take a look next week and see if it's a quick add. If not, we'll make a note about this for future consideration. 🙂

**Comment by kjmartens** [comment]
This is now added in Freeform 3.13.12 🙂
